hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/Federation.md

```diff
@@ -235,7 +235,112 @@ SQL-Server scripts are located in **sbin/FederationStateStore/SQLServer/**.
+  - HashBroadcastPolicyManager
+    - Policy that routes applications via hashing of their queuename, and broadcast resource requests. This picks a HashBasedRouterPolicy for the router and a BroadcastAMRMProxyPolicy for the amrmproxy as they are designed to work together.
+    - We configure to use this policy:
+    ```xml
+      <property>
+         <name>yarn.federation.policy-manager</name>
+         <value>org.apache.hadoop.yarn.server.federation.policies.manager.HashBroadcastPolicyManager</value>
+      </property>
+    ```
+  - HomePolicyManager
+    - Policy manager which uses the UniformRandomRouterPolicy for the Router and HomeAMRMProxyPolicy as the AMRMProxy policy to find the RM.
+    - We configure to use this policy:
+    ```xml
+      <property>
+         <name>yarn.federation.policy-manager</name>
+         <value>org.apache.hadoop.yarn.server.federation.policies.manager.HomePolicyManager</value>
+      </property>
+    ```
+  - PriorityBroadcastPolicyManager
+    - Policy that allows operator to configure "weights" for routing. This picks a PriorityRouterPolicy for the router and a BroadcastAMRMProxyPolicy for the amrmproxy as they are designed to work together.
+    - We configure to use this policy:
+    ```xml
+      <property>
+         <name>yarn.federation.policy-manager</name>
+         <value>org.apache.hadoop.yarn.server.federation.policies.manager.PriorityBroadcastPolicyManager</value>
+      </property>
+    ```
+  - RejectAllPolicyManager
+    - This policy rejects all requests for both router and amrmproxy routing. This picks a RejectRouterPolicy for the router and a RejectAMRMProxyPolicy for the amrmproxy as they are designed to work together.
+    - We configure to use this policy:
+    ```xml
+      <property>
+         <name>yarn.federation.policy-manager</name>
+         <value>org.apache.hadoop.yarn.server.federation.policies.manager.RejectAllPolicyManager</value>
+      </property>
+    ```
+  - UniformBroadcastPolicyManager
+    - It combines the basic policies: UniformRandomRouterPolicy and BroadcastAMRMProxyPolicy, which are designed to work together and "spread" the load among sub-clusters uniformly. This simple policy might impose heavy load on the RMs and return more containers than a job requested as all requests are (replicated and) broadcasted.
+    - We configure to use this policy:
+    ```xml
+      <property>
+         <name>yarn.federation.policy-manager</name>
+         <value>org.apache.hadoop.yarn.server.federation.policies.manager.UniformBroadcastPolicyManager</value>
+      </property>
+    ```
+  - WeightedLocalityPolicyManager
+    - Policy that allows operator to configure "weights" for routing. This picks a LocalityRouterPolicy for the router and a LocalityMulticastAMRMProxyPolicy for the amrmproxy as they are designed to work together.
+    - We configure to use this policy:
```

Comment:
This setting is a little repetitive as we say the same over and over.
Probably we can just give it at the beginning and refer to the name.

Reply:
Thank you for your suggestion! I will improve the documentation.